tests/test_widgets.py

```diff
@@ -5,11 +5,17 @@
 from pyvista import examples
 from pyvista.plotting import system_supports_plotting
 
-NO_PLOTTING = not system_supports_plotting()
 mesh = examples.load_uniform()
 
+# skip all tests if unable to render
+if not system_supports_plotting():
+    pytestmark = pytest.mark.skip
+
+AT_LEAST_VTK_9_1_0 = False
+if pyvista._vtk.VTK9:
+    AT_LEAST_VTK_9_1_0 = pyvista._vtk.vtkVersion().GetVTKMinorVersion() >= 1
```

Comment:
I think we should generalize this too. This pattern will become unmaintainable.  That could be a separate PR, especially given that this is in a test.

Reply:
What about (finally) adding in a vtk version?  For example:
```py
>>> import pyvista
>>> pyvista.vtk_version
(9, 1, 0)
```

Reply:
With dev wheels and some peculiar vtk version choices (paraview??), I think we should go with something closer to an stdlib `version_info`. Namedtuple? A more complex object? Not sure how a dev wheel would fit in this in a way that still makes
```
if pyvista.vtk_version >= (9, 0)
```
work in a sane way.

Reply:
Implemented in 9b631903af5ab40cfaf3113de90b17339f5ba832 as `pyvista.vtk_version_info`.

As far as I can tell, dev wheels will still work with:
```py
pyvista.vtk_version_info >= (9, 0)
```
Provided the base VTK release is 9.0.X****

Reply:
Yeah, I was worried about a potential string-valued micro version breaking comparison with a proper numerical micro version. It's likely that VTK's API only returns a numerical micro anyway. And we're unlikely to test for micro versions.